src/vs/workbench/browser/parts/quickopen/quickOpenController.ts

```diff
@@ -479,6 +483,21 @@ export class QuickOpenController extends WorkbenchComponent implements IQuickOpe
 
 		this.previousValue = prefix;
 
+		// Track active editor before navigation
+		this.previousActiveEditorInput = this.editorService.getActiveEditorInput();
+
+		// Determine if there was a preview editor already open
+		this.previousPreviewEditorInput = null;
+		const activeGroup = this.editorGroupService.getStacksModel().activeGroup;
+		if (activeGroup) {
+			const visiblePreviewEditors = activeGroup.getEditors().filter((input: EditorInput) => {
```

Comment:
good catch on these.  I had not noticed those props earlier and this needed a refactor.